__tests__/index.js

```diff
@@ -252,8 +252,11 @@ describe('Manipulate', () => {
 
   it('End of Time', () => {
     const m = moment(time).endOf('day');
+    const n = new Date(time);
+    n.setHours(23, 59, 59, 999);
```

Comment:
well the case is, `setHours` is mutable function and doesn't return the date. Instead it [returns the milliseconds since epoch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours)

Reply:
Yeah that returns the miliseconds and you compare miliseconds in test ;)

Reply:
I see, I was assuming that the replacement must return a Date object. OK will change it soon